app/mailers/credit_note_mailer.rb

```diff
@@ -16,10 +16,16 @@ def created
       attachments["credit_note-#{@credit_note.number}.pdf"] = file.read
     end
 
+    from_email = if @organization.from_email_enabled?
+      @organization.email
+    else
+      ENV["LAGO_FROM_EMAIL"]
+    end
```

Comment:
I'd move this to the organization model to avoid duplication

```ruby
def from_email_address
    if from_email_enabled?
      email
    else
      ENV["LAGO_FROM_EMAIL"]
    end
end
```

Reply:
done!